descriptors/ModuleDescriptor-template.json

```diff
@@ -4,7 +4,7 @@
   "provides": [
     {
       "id": "users-bl",
-      "version": "3.2",
+      "version": "3.3",
```

Comment:
the same

Reply:
The only change in module descriptor is permissions, so i think i should keep version the same as in MODUSERBL-52